### PR TITLE
Verify file_io.create_dir and file_io.list_directory in GCS test .

### PR DESCRIPTION
**Pull request by anand-c-goog**

### CODE REVIEW

File: tensorflow/tools/gcs_test/python/gcs_smoke.py [hunk 1]
```diff
@@ -48,6 +50,23 @@ def create_examples(num_examples, input_mean):
     examples.append(ex)
   return examples
 
+def create_dir_test():
+  # create_dir
```
Comment (caisq) [inline tensorflow/tools/gcs_test/python/gcs_smoke.py]: nit: Provide a more informative comment.

File: tensorflow/tools/gcs_test/python/gcs_smoke.py [hunk 2]
```diff
@@ -48,6 +50,23 @@ def create_examples(num_examples, input_mean):
...
+  starttime = int(round(time.time() * 1000))
+  dir_name = "%s/tf_gcs_test_%s" % (FLAGS.gcs_bucket_url, starttime)
+  print("Creating dir %s" % dir_name)
+  file_io.create_dir(dir_name)
+  elapsed = int(round(time.time() * 1000)) - starttime
+  print("Created directory in: %d milliseconds" % elapsed)
+  # Check that the directory exists.
+  print("%s directory exists: %s" % (dir_name, file_io.is_directory(dir_name)))
+
+  # List.
```
Comment (caisq) [inline tensorflow/tools/gcs_test/python/gcs_smoke.py]: nit: --> List directory.

File: tensorflow/tools/gcs_test/python/gcs_smoke.py [hunk 3]
```diff
@@ -48,6 +50,23 @@ def create_examples(num_examples, input_mean):
...
+def create_dir_test():
+  # create_dir
+  starttime = int(round(time.time() * 1000))
+  dir_name = "%s/tf_gcs_test_%s" % (FLAGS.gcs_bucket_url, starttime)
+  print("Creating dir %s" % dir_name)
+  file_io.create_dir(dir_name)
+  elapsed = int(round(time.time() * 1000)) - starttime
+  print("Created directory in: %d milliseconds" % elapsed)
+  # Check that the directory exists.
+  print("%s directory exists: %s" % (dir_name, file_io.is_directory(dir_name)))
```
Comment (caisq) [inline tensorflow/tools/gcs_test/python/gcs_smoke.py]: Should this print statement be moved after list_directory succeeds?

File: tensorflow/tools/gcs_test/python/gcs_smoke.py (hunk 1 above)
Comment (anand-c-goog) [inline tensorflow/tools/gcs_test/python/gcs_smoke.py]: Done.

File: tensorflow/tools/gcs_test/python/gcs_smoke.py (hunk 2 above)
Comment (anand-c-goog) [inline tensorflow/tools/gcs_test/python/gcs_smoke.py]: Done.

File: tensorflow/tools/gcs_test/python/gcs_smoke.py (hunk 3 above)
Comment (anand-c-goog) [inline tensorflow/tools/gcs_test/python/gcs_smoke.py]: Updated. LMK if that matches what you were recommending here.

**Review by caisq — APPROVED** [review]

**Comment by caisq** [comment]
@tensorflow-jenkins, test this please.

**Pull request merged**